lens/lotus/api.go

```diff
@@ -37,6 +38,10 @@ type APIWrapper struct {
 	store adt.Store
 }
 
+func (aw *APIWrapper) ChainGetTipSetAfterHeight(ctx context.Context, epoch abi.ChainEpoch, key types.TipSetKey) (*types.TipSet, error) {
+	panic("implement me")
```

Comment:
were killing all these soon, panic seems fine.

Reply:
It does mean that walks against a lotus api will be broken in master. We ought to have a tagged release before this happens.

Alternatively we could fudge it by returning ChainGetTipSetByHeight(height+1). That is no worse than what we are currently doing.